dockerfiles/docker/Dockerfile

```diff
@@ -3,10 +3,7 @@ WORKDIR /go/src/github.com/sourcegraph/lsp-adapter
 COPY . .
 RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-adapter
 
-# The only tag available is latest.
```

Comment:
Cool! TIL about `FROM image@digest`

Reply:
If we ever actually pin to a digest, I would want to see a comment in the dockerfile with the relevant semver for readability.

In practice I think tagging to semver tags is good enough since they should be stable (even though that is not enforced anywhere).

Reply:
@nicksnyder you can include both a tag and a hash for readability:
```dockerfile
FROM node:9@sha256:ca12dd0664cba60c0dee18fb0cfa73336285f0d0ac915d61cb1c8a04aa0f74e6
```

Reply:
Example where we already do this because the image has/had no tags:

https://github.com/sourcegraph/infrastructure/blob/d6bfbd7b8d037c79374eeb132a8afbe063646abc/kubernetes/cmd/sourcegraph-server-gen/internal/dogfood/e2e-bot.go#L38